### PR TITLE
SAMT Wrapper

### PR DESCRIPTION
**Pull request by PascalHonegger**

### CODE REVIEW

**Review by mjossdev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### Advanced
+
+- [Authoring Generators](https://github.com/samtkit/core/wiki/Authoring-Generators)
+- [Architecture](https://github.com/samtkit/core/wiki/Architecture)
+
+## Development Setup
+
+## Contributing
+
+Want to report a bug, contribute code, or improve documentation? Excellent! Simply create
```
Comment (KCreate) [inline README.md:36]: Remove hardcoded linebreaks

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- [Getting Started](https://github.com/samtkit/core/wiki/Getting-Started)
+- [Modeling Concepts](https://github.com/samtkit/core/wiki/Modeling-Concepts)
+- [Visual Studio Code Plugin](https://marketplace.visualstudio.com/items?itemName=samt.samt)
+
+### Advanced
+
+- [Authoring Generators](https://github.com/samtkit/core/wiki/Authoring-Generators)
+- [Architecture](https://github.com/samtkit/core/wiki/Architecture)
+
+## Development Setup
```
Comment (KCreate) [inline README.md:32]: Missing content?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+[![Latest Stable Release on GitHub](https://img.shields.io/github/v/release/samtkit/core?display_name=tag&sort=semver)](https://github.com/samtkit/core/releases/latest)
+[![Total Downloads on GitHub](https://img.shields.io/github/downloads/samtkit/core/total)](https://github.com/samtkit/core/releases/latest)
+[![MIT License](https://img.shields.io/github/license/samtkit/core)](./LICENSE)
+
+</div>
+
+<p align="center">
+  <i>Tired of unreadable OpenAPI YAML files and a plethora of different tools?
+    <br>SAMT is a developer-focussed, extendable and easy-to-learn toolkit for modeling APIs using a business-first approach</i>
```
Comment (KCreate) [inline README.md:13]:
```suggestion
    <br>SAMT is a developer-focused, extendable and easy-to-learn toolkit for modeling APIs using a business-first approach</i>
```

**Review by KCreate — CHANGES_REQUESTED** [review]

**Review by KCreate — APPROVED** [review]
looks good

**Pull request merged**